samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/README.txt

```diff
@@ -1 +1,40 @@
-﻿Test image is from pexels.com: https://www.pexels.com/photo/red-stop-sign-39080/
\ No newline at end of file
+﻿---
```

Comment:
extension should me `md`

Reply:
https://github.com/dotnet/machinelearning-samples/pull/882/commits/fb73c03cbd2ec1e4ca0f747c0e7c6dbe462498c7